analytics/cdk/emr-eks/cdk.context.json

```diff
@@ -0,0 +1,12 @@
+{
+  "availability-zones:account=214783019211:region=eu-west-1": [
+    "eu-west-1a",
+    "eu-west-1b",
+    "eu-west-1c"
+  ],
+  "availability-zones:account=372775283473:region=eu-west-1": [
+    "eu-west-1a",
+    "eu-west-1b",
+    "eu-west-1c"
+  ]
```

Comment:
We should take the account numbers out and document the deployment process in readme.

Reply:
We can either use `cdk.context.json` or use init a stack with the right `env` and pass it to the blueprint